src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -482,7 +482,7 @@
   <Target Name="SetRazorCodeAnalysisProperties" BeforeTargets="RazorCoreCompile" AfterTargets="CopyFilesToOutputDirectory">
     <PropertyGroup Condition=" $(ErrorLog) == $(SonarCompileErrorLog) ">
       <!-- Keep the path so that we could set SonarQubeSetting with sonar.language.roslyn.reportFilePaths -->
-      <RazorSonarCompileErrorLog>$(TargetDir)$(TargetName)$(RazorTargetNameSuffix)$(TargetExt).RoslynCA.json</RazorSonarCompileErrorLog>
+      <RazorSonarCompileErrorLog>$(RazorSonarCompileErrorLog)</RazorSonarCompileErrorLog>
```

Comment:
This change is a no-op; it's assigning `RazorSonarCompilerErrorLog` to itself (and it will always be empty).
The fix suggested by Pavel in the ticket looks like the correct one i.e.
```
<ErrorLog>$(RazorSonarCompileErrorLog)</ErrorLog> 
<RazorCompilationErrorLog>$(RazorSonarCompileErrorLog)</RazorCompilationErrorLog> 
```

As a minimum, there should be a unit test that the expected properties are set when the target is executed.
A full integration test would be nice but more work to set up.